packages/contentful/migrations/crn/workingGroups/20230817113657-add-last-updated-field.js

```diff
@@ -0,0 +1,30 @@
+module.exports.description = 'Adds last updated field';
+
+module.exports.up = function (migration) {
+  const workingGroups = migration.editContentType('workingGroups');
+  workingGroups
+    .createField('lastUpdated')
+    .name('Last Updated')
+    .type('Date')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  workingGroups.changeFieldControl(
+    'lastUpdated',
+    'app',
+    'v97H7wgmtstfxNheYWy0G', // last partial update app
+    {
+      // this is a hack, we dont want to exclude any field
+      // but exclude is required
+      exclude: '"non-existent"',
```

Comment:
I have changed the app definition to make this not be required. 
```suggestion
      exclude: '',
```

Reply:
We probably want to update the app code so that it still works if the parameter is undefined. I think right now it will throw because we call `.split(',)` on the value without checking it is a string.